estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,67 @@
+from odoo import api, models, fields
```

Comment:
Should be sorted alphabetically.
```suggestion
from odoo import api, fields, models
```